### PR TITLE
SAR-4663 updated MultipleVatCheckController to use the new orchestrat…

### PR DESCRIPTION
**Pull request by VLukovski**
…ion service and check the feature switch

### CODE REVIEW

File: it/uk/gov/hmrc/vatsignupfrontend/controllers/principal/MultipleVatCheckControllerISpec.scala [hunk 1]
```diff
@@ -42,106 +46,240 @@ class MultipleVatCheckControllerISpec extends ComponentSpecBase with CustomMatch
...
+          val res = post("/more-than-one-vat-business")(MultipleVatCheckForm.yesNo -> YesNoMapping.option_no)
+
+          res should have(
+            httpStatus(SEE_OTHER),
+            redirectUri(routes.MigrationInProgressErrorController.show().url)
+          )
+        }
+      }
+
+      //      "redirect to business already signed up error page" when {
```
Comment (AlexRimmerHMRC) [inline it/uk/gov/hmrc/vatsignupfrontend/controllers/principal/MultipleVatCheckControllerISpec.scala]: Commented?

File: app/uk/gov/hmrc/vatsignupfrontend/controllers/principal/MultipleVatCheckController.scala [hunk 2]
```diff
@@ -36,7 +37,8 @@ import scala.concurrent.Future
 
 @Singleton
 class MultipleVatCheckController @Inject()(val controllerComponents: ControllerComponents,
-                                           storeVatNumberService: StoreVatNumberService)
+                                           vatNumberOrchestrationService: VatNumberOrchestrationService,
+                                           claimSubscriptionService: ClaimSubscriptionService)
```
Comment (r-melvin) [inline app/uk/gov/hmrc/vatsignupfrontend/controllers/principal/MultipleVatCheckController.scala]: Not needed

File: app/uk/gov/hmrc/vatsignupfrontend/controllers/principal/MultipleVatCheckController.scala [hunk 3]
```diff
@@ -47,49 +49,80 @@ class MultipleVatCheckController @Inject()(val controllerComponents: ControllerC
...
+                      storeVat(vatNumber, isFromBta = false)
+                  }
+              }
+            )
+          case _ =>
+            Future.successful(
+              Redirect(routes.ResolveVatNumberController.resolve())
+            )
+        }
     }
```
Comment (MarjanMubarok) [inline app/uk/gov/hmrc/vatsignupfrontend/controllers/principal/MultipleVatCheckController.scala]: Make the two branches into private defs?

File: test/uk/gov/hmrc/vatsignupfrontend/controllers/principal/MultipleVatCheckControllerSpec.scala [hunk 4]
```diff
@@ -172,7 +176,7 @@ class MultipleVatCheckControllerSpec extends UnitSpec with MockControllerCompone
               redirectLocation(result) shouldBe Some(routes.AlreadySignedUpController.show().url)
             }
           }
-          "redirect to Cannot Sign Up Another Account error page" when {//TODO change test name
+          "redirect to Cannot Sign Up Another Account error page" when { //TODO change test name
```
Comment (MarjanMubarok) [inline test/uk/gov/hmrc/vatsignupfrontend/controllers/principal/MultipleVatCheckControllerSpec.scala]: Is this TODO still relevant?

**Pull request merged**